### ISSUE TITLE
undefined: ghrel.FetchLatestRelease

### ISSUE THREAD

**Issue by DamianSkrzypczak** [gopasspw/gopass#1199]
### Summary
Error `undefined: ghrel.FetchLatestRelease` appears during `go get` installation.

### Steps To Reproduce
Attempt to install `gopass` using `go get` with go version 1.13.1
in project outside `$GOPATH` which have `go.mod` 

### Expected behavior
Installations successes

### Environment
go version 1.13.1
GO111MODULE=on
Installation happens for project that uses `go.mod`

- OS: Manjaro
- OS version: 4.19.78-1-MANJARO
- gopass Version: 1.8.6
- Installation method: `go get` (`go get -u` checked as well)

### Additional context
After brief check I see that `master` branch from github.com/dominikschulz/github-releases/ghrel contains `FetchLatestRelease` but tag `v0.0.1` doesn't. Release `github-releases` `v0.0.2` should fix it. @dominikschulz

**Comment by DamianSkrzypczak** [comment]
Console output:
```bash
> go get github.com/gopasspw/gopass
go: finding github.com/xrash/smetrics latest
go: finding golang.org/x/crypto latest
go: finding github.com/martinhoefling/goxkcdpwgen latest
go: finding github.com/muesli/crunchy latest
go: finding github.com/alecthomas/binary latest
go: finding golang.org/x/sys latest
go: finding github.com/kevinburke/ssh_config latest
go: finding github.com/jbenet/go-context latest
go: finding golang.org/x/net latest
go: finding github.com/jsimonetti/pwscheme latest
go: finding github.com/kballard/go-shellquote latest
go: finding github.com/armon/go-metrics latest
go: finding github.com/nsf/termbox-go latest
go: finding golang.org/x/time latest
go: finding github.com/mitchellh/go-ps latest
go: finding github.com/muesli/goprogressbar latest
go: finding github.com/skip2/go-qrcode latest
# github.com/gopasspw/gopass/pkg/updater
go/pkg/mod/github.com/gopasspw/gopass@v1.8.6/pkg/updater/update.go:94:10: undefined: ghrel.FetchLatestRelease
```

**Comment by DamianSkrzypczak** [comment]
Is there is any progress on this issue done so far?

**Comment by acheron2302** [comment]
It seem like changing the the go.mod `github.com/dominikschulz/github-releases v0.0.1` to `github.com/dominikschulz/github-releases v0.0.0-20171016132906-7635b43b2447` fix the problem

**Comment by acheron2302** [comment]
The problem is because in v0.0.1 He remove the function FetchLatestRelease but go mod fetch the latest release so when update to v.0.0.1 it get that error. I think we should change from FetchLatestRelease function to FetchLatestStableRealease function

**Comment by acheron2302** [comment]
#1237

**Comment by acheron2302** [comment]
For now the temporary solution is just to use git clone this at $GOPATH/src/gopasspw and then make install at there.

**Comment by dominikschulz** [comment]
Should be fixed by #1273.
